pkg/rpc/client/rpc_test.go

```diff
@@ -0,0 +1,1502 @@
+			},
+		},
+		{
+			name: "getblockheader_verbose_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetBlockHeaderVerbose(util.Uint256{})
+			},
+		},
+		{
+			name: "getblocksysfee_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetBlockSysFee(1)
+			},
+		},
+		{
+			name: "getclaimable_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetClaimable("")
+			},
+		},
+		{
+			name: "getconnectioncount_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetConnectionCount()
+			},
+		},
+		{
+			name: "getcontractstate_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetContractState(util.Uint160{})
+			},
+		},
+		{
+			name: "getnep5balances_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetNEP5Balances(util.Uint160{})
+			},
+		},
+		{
+			name: "getnep5transfers_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetNEP5Transfers("")
+			},
+		},
+		{
+			name: "getrawtransaction_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetRawTransaction(util.Uint256{})
+			},
+		},
+		{
+			name: "getrawtransaction_verbose_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetRawTransactionVerbose(util.Uint256{})
+			},
+		},
+		{
+			name: "getstorage_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetStorage(util.Uint160{}, []byte{})
+			},
+		},
+		{
+			name: "gettransactionheight_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetTransactionHeight(util.Uint256{})
+			},
+		},
+		{
+			name: "gettxoutput_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetTxOut(util.Uint256{}, 0)
+			},
+		},
+		{
+			name: "getunclaimed_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetUnclaimed("")
+			},
+		},
+		{
+			name: "getunspents_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetUnspents("")
+			},
+		},
+		{
+			name: "invokefunction_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.InvokeFunction("", "", []smartcontract.Parameter{})
+			},
+		},
+		{
+			name: "invokescript_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.InvokeScript("")
+			},
+		},
+		{
+			name: "sendrawtransaction_invalid_params_error",
+			invoke: func(c *Client) (interface{}, error) {
+				return nil, c.SendRawTransaction(&transaction.Transaction{
+					Type:       0,
+					Version:    0,
+					Data:       nil,
+					Attributes: nil,
+					Inputs:     nil,
+					Outputs:    nil,
+					Scripts:    nil,
+					Trimmed:    false,
```

Comment:
Why is this one expanded? I thought `&transaction.Transaction{}` should be enough?